Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -205,6 +213,20 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         bodyField.inputAccessoryView = actions
     }
 
+    func updateSubmitButtonState() {
+        if titleText == nil {
```

Comment:
Maybe combine these 2 ifs with an or ( `||`) or all three with a ternary. Something like

 navigationItem.rightBarButtonItem?.isEnabled = ( titleText == nil || bodyText == nil ) ? false : true

Reply:
Well the `false : true` is unnecessary, of course. Although you should then flip the result. 😊

Reply:
Hahahahah! Yes correct on both points. Totally forget the ternary

Reply:
Ok. I can change this around. How should I resubmit? I close and then do a new PR right?

Reply:
@wayni208 no need to close. Once you make a new commit to your branch it will update here

Reply:
There we go! Thanks for the feedback and direction on that function. Such a more elegant way to get it done.

Reply:
Awesome! Was about to say you don't have to reopen every time. 💯